packages/build-ui/src/pages/agits/_id/management/components/common_tables.rs

```diff
@@ -0,0 +1,246 @@
+                                        span { class: "pl-4 text-xs", "24h" }
+                                    }
+                                    div {
+                                        class: "flex flex-col",
+                                        span {
+                                            class: "text-red-500",
+                                            {asset.price_change_7d.to_string()}
+                                        }
+                                        span { class: "pr-4 text-xs", "7d" }
+                                    }
+                                }
+                            }
+                            td {
+                                class: "px-2 py-2 sm:px-4 sm:py-3 whitespace-nowrap",
+                                div {
+                                    class: "flex flex-col",
+                                    span { {asset.volume.to_string()} }
+                                    span { class: "text-xs text-gray-500", {asset.volume_usd.to_string()} }
+                                }
+                            }
+                            td {
+                                class: "px-2 py-2 sm:px-4 sm:py-3 whitespace-nowrap",
+                                div {
+                                    class: "flex flex-col",
+                                    span { {asset.royalty.to_string()} }
+                                    span { class: "text-xs text-gray-500", {asset.royalty_usd.to_string()} }
+                                }
+                            }
+                            td {
+                                class: "px-2 py-2 sm:px-4 sm:py-3 whitespace-nowrap",
+                                {asset.status.to_string()}
+                            }
+                            td {
+                                class: "px-2 py-2 sm:px-4 sm:py-3 whitespace-nowrap",
+                                div {
+                                    class: "relative",
+                                    button {
+                                        class: "text-gray-400 hover:text-white",
+                                        onclick: move |_| {
+                                            active_dropdown.set(
+                                                if is_dropdown_open { None } else { Some(index) }
+                                            ); // Toggle dropdown: open if closed, close if open
+                                        },
+                                        validations::Extra { class: "[&>circle]:stroke-white", height: 18 }
+                                    }
+                                    // Render dropdown if this row's dropdown is active
+                                    if is_dropdown_open {
+                                        div {
+                                            class: "absolute right-0 mt-2 w-48 bg-black border border-green-500 rounded shadow-lg z-10",
```

Comment:
```suggestion
                                        div {
                                            class: "absolute right-0 mt-2 w-48 bg-black border border-green-500 rounded shadow-lg z-10 hidden aria-dropdown-open:block",
                                            "aria-dropdown-open": is_dropdown_open,
```

Dioxus re-compile whenever change rust-related part.
If you utilize `aria-*` of tailwindcss, you can minimize code making recompiling.